src/shared/sleep-config.c

```diff
@@ -141,6 +142,147 @@ int read_battery_capacity_percentage(void) {
         return battery_capacity;
 }
 
+static int get_battery_identifiers(char **ret_identifiers) {
+        _cleanup_free_ char *bat_manufacturer = NULL, *bat_model_name = NULL, *bat_serial_number = NULL;
+        _cleanup_free_ char *p = NULL;
```

Comment:
merge in the declaration above

Reply:
Yeah. I moved to next line to keep width less than 109. Recently saw that in .vimrc.

Reply:
you can just go to the new line, without having to re-add the type

Reply:
ok